doc/dev_guide/internals/box_protocol.rst

```diff
@@ -739,12 +739,8 @@ The body is optional and can contain two items:
 IPROTO_TIMEOUT is an optional timeout (in seconds). After it expires,
 the transaction will be rolled back automatically.
 
-.. // TODO: add link to transaction isolation docs once they're ready
-
-IPROTO_TXN_ISOLATION is the transaction isolation level. It can take
-the following values:
-
-.. // TODO: provide links to level descriptions
+IPROTO_TXN_ISOLATION is the :ref:`transaction isolation level <txn_mode_mvcc-options>`.
+It can take the following values:
 
```

Comment:
It's not very clear what a global default level is? Or why should I choose the automatic level detection?

Reply:
I'd wouldn't expand these brief descriptions because it's just a reference and is already linked to the page with conceptual explanation.